internal/cluster/cluster_operations.go

```diff
@@ -356,5 +357,23 @@ func ImportCluster(ctx context.Context, data *schema.ResourceData, apiClient *cl
 		flattenedHostSpecs = append(flattenedHostSpecs, *FlattenHost(hostObj))
 	}
 	_ = data.Set("host", flattenedHostSpecs)
+
+	//get all domains and find our cluster to set the "domain_id" attribute, because
+	// cluster API doesn't provide parent domain ID.
```

Comment:
That's odd! Please file a bug on the REST API.

Reply:
Done (see relevant non-public channels)